### PR TITLE
deps/cacert: Update to 2024-11-26

### PR DESCRIPTION
**Pull request by eschnett**
@giordano

### CODE REVIEW

File: deps/libgit2.version [hunk 1]
```diff
@@ -11,4 +11,4 @@ LIBGIT2_SHA1=d74d491481831ddcd23575d376e56d2197e95910
 # The versions of cacert.pem are identified by the date (YYYY-MM-DD) of their changes.
 # See https://curl.haxx.se/docs/caextract.html for more details.
 # Keep in sync with `stdlib/MozillaCACerts_jll/Project.toml`.
```
Comment (giordano) [inline deps/libgit2.version:13]: 🙂 You can also follow [this checklist](https://docs.julialang.org/en/v1/devdocs/build/build/#Update-the-version-number-of-a-dependency) as a reference.

Comment (eschnett) [reply deps/libgit2.version:13]: Thanks!

File: deps/libgit2.version [hunk 2]
```diff
@@ -11,4 +11,5 @@ LIBGIT2_SHA1=d74d491481831ddcd23575d376e56d2197e95910
...
-MOZILLA_CACERT_VERSION := 2024-03-11
+MOZILLA_CACERT_VERSION := 2024-11-26
+
```
Comment (giordano) [inline deps/libgit2.version:15]:
```suggestion
```

Comment (eschnett) [reply deps/libgit2.version:15]: I needed to add that to trigger a rebuild, there was a spurious build failure.

Comment (eschnett) [reply deps/libgit2.version]: Applied.

**Pull request merged**